src/Microsoft.DocAsCode.Metadata.ManagedReference/Transform/TripleSlashCommentTransform.xsl

```diff
@@ -111,6 +111,34 @@
     </xsl:if>
   </xsl:template>
 
+  <xsl:template match="note">
+    <xsl:variable name="type">
```

Comment:
i meant that the output should be consistent, at html level?

---
In reply to: [96158289](https://github.com/dotnet/docfx/pull/1171#discussion_r96158289) [](ancestors = 96158289)